c7n/resources/org.py

```diff
@@ -191,8 +191,133 @@ def parse_query(self):
                 self.account_config["org-account-role"] = "AWSControlTowerExecution"
 
 
-@OrgUnit.filter_registry.register('org-unit')
-@OrgAccount.filter_registry.register('org-unit')
+@OrgUnit.filter_registry.register("policy")
+@OrgAccount.filter_registry.register("policy")
+class PolicyFilter(ListItemFilter):
+    schema = type_schema(
+        "policy",
+        required=["policy-type"],
+        **{
+            "policy-type": {"enum": OrgPolicy.policy_types},
+            "inherited": {"type": "boolean"},
+            "attrs": {"$ref": "#/definitions/filters_common/list_item_attrs"},
+            "count": {"type": "number"},
+            "count_op": {"$ref": "#/definitions/filters_common/comparison_operators"},
+        },
+    )
+
+    permissions = ("organizations:ListRoots", "organizations:ListPoliciesForTarget")
+
+    annotate_items = True
+    annotation_key = "c7n:PolicyMatches"
+    target_policies = None
+    ou_root = None
+    client = None
+
+    def process(self, resources, event):
+        self.client = local_session(self.manager.session_factory).client("organizations")
+        if self.data.get("inherited") and self.manager.type == "org-account":
+            # Get ou account hierarchy / parents
+            hierarchy_manager = self.manager.get_resource_manager(
+                "org-account", {"filters": ["org-unit"]}
+            )
+
+            ou_assembly = hierarchy_manager.filters[0]
+            ou_assembly.ou_map = {
+                ou["Id"]: ou for ou in self.manager.get_resource_manager("org-unit").resources()
+            }
+            ou_assembly.process_accounts(resources, event)
+            # also initialize root for accounts as we dont store it as a parent.
+            self.ou_root = self.client.list_roots()["Roots"][0]
+        self.target_policies = {}
+        return super().process(resources, event)
+
+    def get_targets(self, resource):
+        if not self.data.get("inherited"):
+            yield resource["Id"]
+            return
+
+        # return self, parents, root
```

Comment:
stray comment?

Reply:
bitrotted doc when it was a data structure, but moved it to a simple string, so indeed stray.